.travis.yml

```diff
@@ -135,6 +131,7 @@ base_postgres: &postgres_default
   install:
     - python setup.py install_egg_info
     - pip install -U -e ".[dev]"
+    - sentry init
```

Comment:
Are these changes related?

Reply:
Yep, sentry init can't be called twice - the 2nd time will error in CI because it wants an interactive y/n to overwrite the config file.

It also makes more sense to explicitly execute it after installing sentry anyways.

Reply:
Previously this was in the Makefile but in GHA we use devservices to spin up services, and so the actual testing step which uses make did init a 2nd time.